### DISCUSSION TITLE
Maintain nvim-tree's window dimentions when closing opened buffers

### DISCUSSION THREAD

**Discussion by SephVelut** [nvim-tree/nvim-tree.lua#1733] [Q&A] [answered]
If I have nvim-tree open along with 2 or more vim buffers open, when I close one of the buffers, nvim-tree's window width gets maximized. So then I have to resize/clean all the windows back up or close and re-open nvim-tree. Anyway to avoid this behavior?

**Comment by gegoune** [comment, marked as answer]
Vaguely related https://github.com/nvim-tree/nvim-tree.lua/issues/149
